gemma-rest/src/main/antlr4/ubic/gemma/rest/util/args/FilterArg.g4

```diff
@@ -0,0 +1,43 @@
+grammar FilterArg;
+
+CONJUNCTION: 'and' | 'AND';
+DISJUNCTION: 'or' | 'OR';
+
+GE: '>';
+GEQ: '>=';
+LE: '<';
+LEQ: '<=';
+EQ: '=';
+NEQ: '!=';
+LIKE: 'like';
+IN: 'in';
+
+FLOAT: [0-9]+ '.' [0-9]+ 'f';
+DOUBLE: [0-9]+ '.' [0-9]+;
+LONG: [0-9]+ 'l';
+INTEGER: [0-9]+;
```

Comment:
We need more tests for number formatting, or simply remove these tokens and treat them as strings.